torch/csrc/jit/tensorexpr/registerizer.h

```diff
@@ -0,0 +1,354 @@
+#pragma once
+#include <c10/core/ScalarType.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <vector>
+
+#include <torch/csrc/jit/tensorexpr/hash_provider.h>
+#include <torch/csrc/jit/tensorexpr/ir_mutator.h>
+#include <torch/csrc/jit/tensorexpr/ir_simplifier.h>
+#include <torch/csrc/jit/tensorexpr/ir_visitor.h>
+
+namespace torch {
+namespace jit {
+namespace tensorexpr {
+
+bool exprEquals(const Expr* A, const Expr* B) {
```

Comment:
It will be great to add a comment section explaining what Registerizer pass is. The comment in the commit message is perfectly fine.